packages/database/src/migrations/20240213024728-AddReportLatestDataParametersColumn-modifies-schema.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.addColumn('report', 'latest_data_parameters', {
+    type: 'json',
```

Comment:
We tend to use `jsonb` in tupaia. `json` seems pretty fine in this context, but maybe we wanna stick with `jsonb` for consistency? Looking into the differences:
- JSONB writes a bit slower, but reads faster
- There are a few more operators available for JSONB
- JSONB doesn't preserve key order (mildly annoying at times :sweat_smile:)